SocialCareCaseViewerApi.Tests/V1/Helpers/TestHelpers.cs

```diff
@@ -46,6 +50,19 @@ public static CaseNote CreateCaseNote()
                 .RuleFor(c => c.CreatedByName, f => f.Person.FirstName);
         }
 
+        public static CaseNoteResponse CreateCaseNoteResponse(CaseNote? caseNote)
+        {
+            return new Faker<CaseNoteResponse>()
+                .RuleFor(c => c.RecordId, f => caseNote?.CaseNoteId ?? f.Lorem.Paragraph())
```

Comment:
I think this should be `f.UniqueIndex.ToString()`?

Reply:
Thank you! Nice catch :)